src/editor/nodes/ModelNode.js

```diff
@@ -351,26 +357,23 @@ export default class ModelNode extends EditorNodeMixin(Model) {
     return this;
   }
 
-  prepareForExport(ctx) {
+  prepareForExport() {
     super.prepareForExport();
     this.addGLTFComponent("shadow", {
       cast: this.castShadow,
       receive: this.receiveShadow
     });
 
-    // TODO: Support exporting more than one active clip.
-    if (this.activeClip) {
-      const activeClipIndex = ctx.animations.indexOf(this.activeClip);
-
-      if (activeClipIndex === -1) {
-        throw new Error(
-          `Error exporting model "${this.name}" with url "${this._canonicalUrl}". Animation could not be found.`
-        );
-      } else {
-        this.addGLTFComponent("loop-animation", {
-          activeClipIndex: activeClipIndex
-        });
+    this.addGLTFComponent(
+      "loop-animation",
+      {
+        clip: this.activeClipNames.join(),
```

Comment:
Rather than relying on the clip name, I'd rather we focus on exporting an array of clip indices and modify Hubs to read those indices.

So it'd look something like:
```js
if (this.activeClips.length > 0) {
  const activeClipIndices = this.activeClips.map((clip) => {
    const index = ctx.animations.indexOf(clip);

    if (index === -1) {
      throw new Error(`Error exporting model "${this.name}" with url "${this._canonicalUrl}". Animation could not be found.`);
    }

    return index;
  });


  this.addGLTFComponent("loop-animation", { activeClipIndices });
}
```

Reply:
Updated